### DISCUSSION TITLE
posts/grafana-loki/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#209] [Comments]
# posts/grafana-loki/

Documentation for Techno Tim YouTube videos for all your copy pasta needs!

https://technotim.live/posts/grafana-loki/

**Comment by mongo73** [comment]
Followed the guide and got this error inside of Grafana/Loki when adding sources

Anyone else had issues like this?  I'm still troubleshooting just thougth I'd ask.

Unable to fetch labels from Loki (Failed to call resource), please check the server logs for more details

Thank you for guide and several of your others there very helpful

**Reply by mongo73** [reply]
Please disregard. I had made a typo in the loki-config.yml file.

**Comment by Atrko** [comment]
Not sure if it's okay to post the error message and ask here, anyway, here it is

I've been following along the steps but I am having trouble stand up the containers, any pointers?

ERROR: for 97f9d034dcc7_docker_volumes_promtail_1  Cannot start service promtail: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/var/lib/docker/containers/44fbbe7a812a481989ecc3980884b8
Recreating a009164fe60b_docker_volumes_grafana_1  ... error
s path detected -- refusing to operate on /etc/resolv.conf: unknown

ERROR: for a009164fe60b_docker_volumes_grafana_1  Cannot start service grafana: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error
Recreating 0e3e7eb50676_docker_volumes_loki_1     ... error
f19e7c4916b4a4d19a2a1917b9d4aec5/resolv.conf" to rootfs at "/etc/resolv.conf": possibly malicious path detected -- refusing to operate on /etc/resolv.conf: unknown

ERROR: for 0e3e7eb50676_docker_volumes_loki_1  Cannot start service loki: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/var/lib/docker/containers/a1d0c2da201d08d84bb7490e814ec53fd48f90eef7847b84c649849e3306926f/resolv.conf" to rootfs at "/etc/resolv.conf": possibly malicious path detected -- refusing to operate on /etc/resolv.conf: unknown

ERROR: for promtail  Cannot start service promtail: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/var/lib/docker/containers/44fbbe7a812a481989ecc3980884b8b973acefc0ff70d9b3313487222b563a32/resolv.conf" to rootfs at "/etc/resolv.conf": possibly malicious path detected -- refusing to operate on /etc/resolv.conf: unknown

ERROR: for grafana  Cannot start service grafana: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/var/lib/docker/containers/16355e1cba410e570055e897ccd16227f19e7c4916b4a4d19a2a1917b9d4aec5/resolv.conf" to rootfs at "/etc/resolv.conf": possibly malicious path detected -- refusing to operate on /etc/resolv.conf: unknown

**Comment by Atrko** [comment]
I follow up the steps util containers are up
CONTAINER ID   IMAGE                    COMMAND                  CREATED         STATUS         PORTS                    NAMES
63e6b605e17a   grafana/loki:2.4.0       "/usr/bin/loki -conf…"   3 minutes ago   Up 3 minutes   0.0.0.0:3100->3100/tcp   docker_loki_1
fd2c9e184445   grafana/grafana:latest   "/run.sh"                3 minutes ago   Up 3 minutes   0.0.0.0:3000->3000/tcp   docker_grafana_1
cd5cf988bb52   grafana/promtail:2.4.0   "/usr/bin/promtail -…"   3 minutes ago   Up 3 minutes                            docker_promtail_1

I went on to the ip address of the server and port 3100 for loki. The server returned "This site can’t be reached". Any pointers?
